Sources/ProjectSpec/TargetSource.swift

```diff
@@ -28,17 +28,59 @@ public struct TargetSource: Equatable {
         }
     }
 
-    public enum BuildPhase: String {
+    public enum BuildPhase: Equatable {
         case sources
         case headers
         case resources
+        case copyFiles(CopyFilesSettings)
         case none
+        // Not currently exposed as selectable options, but used internally
+        case frameworks
```

Comment:
indenting

Reply:
@yonaskolb I’m not sure what you are referring to here. Do you want a newline before the comment, the comment to be indented differently, or something else?

Reply:
Hmm, it's not showing up now. I swear `case frameworks` looked indented differently before